react/tic-tac-toe/src/redux/users/actions.js

```diff
@@ -22,6 +22,11 @@ const actionsCreators = {
     } else {
       dispatch({ type: actions.LOGIN_FAILURE });
     }
+  },
+  logout: () => dispatch => {
+    localStorage.removeItem('token');
+    api.setHeaders(defaultHeaders);
```

Comment:
Why you need to do this here 👀 ? You already have your headers set in your api.

Reply:
to remove the token header